tileview/src/main/java/com/qozix/tileview/markers/MarkerLayout.java

```diff
@@ -120,15 +138,25 @@ protected void onMeasure( int widthMeasureSpec, int heightMeasureSpec ) {
       View child = getChildAt( i );
       if( child.getVisibility() != GONE ) {
         MarkerLayout.LayoutParams layoutParams = (MarkerLayout.LayoutParams) child.getLayoutParams();
-        // get anchor offsets
-        float widthMultiplier = (layoutParams.anchorX == null) ? mAnchorX : layoutParams.anchorX;
-        float heightMultiplier = (layoutParams.anchorY == null) ? mAnchorY : layoutParams.anchorY;
         // actual sizes of children
         int actualWidth = child.getMeasuredWidth();
         int actualHeight = child.getMeasuredHeight();
-        // offset dimensions by anchor values
+        // get anchor offsets
+        float widthMultiplier = (layoutParams.anchorX == null) ? mAnchorX : layoutParams.anchorX;
+        float heightMultiplier = (layoutParams.anchorY == null) ? mAnchorY : layoutParams.anchorY;
+        // offset dimensions by anchor values (default typeAnchor = 0)
         float widthOffset = actualWidth * widthMultiplier;
         float heightOffset = actualHeight * heightMultiplier;
+        // handle typeAnchorX options other than 0
+        switch (layoutParams.typeAnchorX) {
```

Comment:
no need to recompute what was already figured on lines 148 and 149, may as well add that as the `default` case to your switch statements

Reply:
Reasonable, I'll fix that.